### PR TITLE
Use fsevents 0.3.7 to avoid segmentation faults

### PR DESCRIPTION
**Pull request by pmq20**
cf.

* https://github.com/strongloop/fsevents/pull/74
* https://github.com/strongloop/fsevents/issues/52

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "sinon-chai": "^2.6.0"
   },
   "optionalDependencies": {
-    "fsevents": "^0.3.1"
+    "fsevents": ">= 0.3.7 < 1"
```
Comment (fengmk2) [inline package.json]: `^0.3.7`

**Comment by es128** [comment]
This is unnecessary. `^0.3.1` will already install the latest 0.3.x automatically, and the only thing existing projects need to do is `npm update`.

**Comment by paulmillr** [comment]
Doesn't bring any disadvantages tho!

**Comment by nickdesaulniers** [comment]
if you can publish a new version number, folks reinstalling watchify will get unblocked, see: https://github.com/substack/watchify/issues/257#issuecomment-132843293

**Comment by es128** [comment]
@nickdesaulniers shouldn't matter. `npm update` (or at worst `rm -rf node_modules && npm i`) should be sufficient to pull in latest fsevents.

master branch is failing tests in CI currently, want to resolve before the next publish

**Comment by nickdesaulniers** [comment]
yep, thanks!

**Comment by es128** [comment]
> Doesn't bring any disadvantages tho!

Turns out it does. It confuses *lots* of users thinking they need to wait for us to publish rather than just updating deps and letting semver do its thing.

**Pull request merged**